semeio/jobs/overburden_timeshift/ots.py

```diff
@@ -50,10 +50,13 @@ def ots_load_params(input_file):
 
 
 def write_surface(vintage_pairs, ts, output_dir, type_str):
```

Comment:
Pathlib??

Shouldn't we use pathlib for stuff like this?

```suggestion
def write_surface(vintage_pairs, ts, output_dir, type_str):
    output_path = Pathlib(output_dir + type_str)
    if not output_path.is_dir():
        output_path.mkdir()
    …
        ts[iv].to_file((output_dir / f"ots_{d0}_{d1}.irap"))    
```

Reply:
🙇